README.md

```diff
@@ -2,15 +2,17 @@
 
 ## Pre-requisites
 - [Install](https://github.com/operator-framework/operator-sdk/blob/v0.17.x/doc/user/install-operator-sdk.md) operator-sdk
-  v0.17.0
-- The operator is written using operator-sdk [v0.17.0](https://github.com/operator-framework/operator-sdk/releases/tag/v0.17.0)
+  v0.17.1
+- The operator is written using operator-sdk [v0.17.1](https://github.com/operator-framework/operator-sdk/releases/tag/v0.17.1)
   and has the same [pre-requisites](https://github.com/operator-framework/operator-sdk/tree/v0.17.x#prerequisites) as it
   does.
+- Instructions assume that the user is using [Podman](https://podman.io/) container engine. Podman can be made default 
```

Comment:
I don't see why this should be done instead of the `--image-builder` arg, but if others are okay with it I am too

Reply:
@PratikMahajan my main reason to suggest not including this was to highlight that we don't support use cases where users have both podman and docker. However if that causes us to run into issues, I am fine with using `--image-builder`.